projects/explorer/src/app/views/accounts/account/account.module.ts

```diff
@@ -1,4 +1,4 @@
-import { MaterialModule } from '../../../views/material.module';
+import { MaterialModule } from '../../material.module';
```

Comment:
@YasunoriMATSUOKA 
Is it a deliberate fix?

Reply:
@mkXultra 
It seems that this was automatically corrected when the directory was moved. The useless description in the original file path specification has been erased, but I thought it pointed to the same file path as a result and was fine with it as it was.